src/app/team/serviceForm/whoIsThisClothingFor/options.js

```diff
@@ -0,0 +1,34 @@
+export const options = [
+  {
+    label: 'Men',
+    wearerAge: ['men', 'adults'],
+    gender: 'male',
+  },
+  {
+    label: 'Women',
+    wearerAge: ['women', 'adults'],
+    gender: 'female',
+  },
+  {
+    label: 'Girls',
+    wearerAge: ['girls', 'children'],
+    gender: 'female',
```

Comment:
Gender should only be added for Men and Women but not Boys and Girls, as otherwise with the current filters, a user that's an adult male could be referred to a location that has clothes only for Women and Boys.